src/main/java/me/jellysquid/mods/sodium/client/gui/options/Option.java

```diff
@@ -1,11 +1,59 @@
 package me.jellysquid.mods.sodium.client.gui.options;
 
+import me.jellysquid.mods.sodium.client.SodiumClientMod;
 import me.jellysquid.mods.sodium.client.gui.options.control.Control;
 import me.jellysquid.mods.sodium.client.gui.options.storage.OptionStorage;
 import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+
 import java.util.Collection;
 
-public interface Option<T> {
+public interface Option<T>{
+    ResourceLocation DEFAULT_ID = new ResourceLocation(SodiumClientMod.MODID, "empty");
```

Comment:
I would prefer these be declared as public static final fields in their own class to avoid cluttering the interface. Same for the other two classes.

Reply:
what about DEFAULT_ID? is needed to perform the default method on interface?

Reply:
Leave that one for now, but I might make `getId` nullable instead afterward.